libs/wingc/src/jsify/snapshots/allow_type_def_before_super.snap

```diff
@@ -125,16 +120,8 @@ class $Root extends $stdlib.std.Resource {
               })
             `;
           }
-          _toInflight() {
-            return `
-              (await (async () => {
-                const BazClient = ${Baz._toInflightType()};
-                const client = new BazClient({
-                });
-                if (client.$inflight_init) { await client.$inflight_init(); }
-                return client;
-              })())
-            `;
+          _liftedState() {
+            return { ...(super._liftedState?.() ?? {}) };
           }
```

Comment:
Since this is a no-op maybe we can omit this method if there is no state to lift...

Reply:
Sure, yeah a little bit less code shouldn't hurt

(I'd honestly like it this to be hard contract / it's easier debugging our system when code fails fast when some contract isn't satisfied -- unfortunately this isn't an option here because we allow you to extend JSII classes, which don't have `_liftedState` methods.)